### PR TITLE
feat: add CDC PLACES tract-level ingestion (#75)

### PR DESCRIPTION
**Pull request by William-Hill**
## Summary
- Add `cdc_places_tract_health` Dagster asset fetching ~83,500 census tracts from CDC PLACES SODA API (`cwsq-ngmh.json`)
- Extract shared `_fetch_places_measures()` and `_parse_row()` helpers
- Refactor existing county asset to use shared helpers (DRY)
- Register asset and add quarterly schedule
- Part 1 of #75

## Test plan
- [x] `pytest dagster/tests/test_cdc_places_asset.py -v` — 8 tests pass
- [x] `pytest dagster/tests/ -v` — 293 tests pass (no regressions)
- [ ] Manual: `dagster dev` shows new asset in UI

## Summary by CodeRabbit

* **New Features**
  * Added census tract–level CDC PLACES health ingestion as a new asset with paginated fetches, batched upserts, metadata and improved observability.
  * Added a quarterly automated schedule to refresh tract-level health metrics.

* **Tests**
  * Added tests validating the new tract-level asset, endpoint URLs, and row-parsing behaviors (including edge cases).

* **Documentation**
  * Added a detailed roadmap for race‑disaggregated health estimates using CDC PLACES and Census data.

### CODE REVIEW

File: dagster/tests/test_cdc_places_asset.py [hunk 1]
```diff
@@ -28,3 +31,25 @@ def test_cdc_measures_non_empty():
...
+
+
+def test_cdc_places_tract_url_is_tract_endpoint():
+    """Tract URL should point to the census tract SODA endpoint."""
+    assert "cwsq-ngmh" in CDC_PLACES_TRACT_URL
+
+
+def test_cdc_places_county_url_is_county_endpoint():
+    """County URL should point to the county SODA endpoint."""
+    assert "swc5-untb" in CDC_PLACES_URL
```
[the remark this answers is not in the document]
Comment (William-Hill) [reply dagster/tests/test_cdc_places_asset.py]: Fixed in b98bebac — added _parse_row unit tests covering valid rows, missing FIPS, missing data_value, and non-numeric data_value.

File: dagster/d4bl_pipelines/assets/apis/cdc_places.py [hunk 2]
```diff
@@ -273,3 +273,213 @@ async def cdc_places_health(
...
+        "state_fips": state_fips,
+        "year": int(row.get("year", 0)),
+        "measure": MEASURE_NAMES.get(measure, measure.lower()),
+        "category": MEASURE_CATEGORIES.get(measure, "other"),
+        "value": value,
+        "dvt": dvt,
+        "low_cl": low_cl,
+        "high_cl": high_cl,
+        "pop": pop,
+    }
```
[the remark this answers is not in the document]
Comment (William-Hill) [reply dagster/d4bl_pipelines/assets/apis/cdc_places.py]: Fixed in 48f4dbef — year is now validated early in _parse_row; rows with missing or non-numeric year are skipped.

File: dagster/tests/test_cdc_places_asset.py [hunk 3]
```diff
@@ -28,3 +32,72 @@ def test_cdc_measures_non_empty():
...
+
+
+def test_parse_row_invalid_value():
+    """_parse_row should return None for non-numeric data_value."""
+    row = {
+        "countyfips": "17031",
+        "data_value": "not_a_number",
+        "measureid": "DIABETES",
+    }
+    assert _parse_row(row, fips_field="countyfips") is None
```
[the remark this answers is not in the document]
Comment (William-Hill) [reply dagster/tests/test_cdc_places_asset.py]: Fixed in 48f4dbef — added test_parse_row_valid_tract (locationid/locationname path) and test_parse_row_missing_year.
